server/controllers/MiscController.js

```diff
@@ -120,6 +120,13 @@ class MiscController {
       Logger.error('Invalid user in authorize')
       return res.sendStatus(401)
     }
+
+    // When authorizing an API token and forward auth is enabled then validate user using forward auth headers
+    if (global.ForwardAuth.Enabled && !this.auth.validateForwardAuthUser(req)) {
+      Logger.error(`[MiscController] Authorize token with forward auth enabled failed for user "${req.user.username}"`)
+      return res.sendStatus(403)
+    }
+
```

Comment:
This is the code that would break the mobile app but I'm realizing now that we probably don't need to return an error code here.
So if forward auth is enabled and the user does not match then just proceed with the user from the token.

Reply:
Just removed that return and changed the error log to a warning.